src/Elmish/Hooks/Type.purs

```diff
@@ -102,6 +108,49 @@ mkHook name mkDef =
 withHooks :: Hook ReactElement -> ReactElement
 withHooks (Hook hook) = hook identity
 
+-- | When there is only one hook, it might make more sense to invoke it with
+-- | continuation-passing style. This helper makes that easier, accepting a
+-- | `render` callback.
+-- |
+-- | ```purs
+-- | view :: ReactElement
+-- | view = withHook (useState "") \(name /\ setName) ->
+-- |   H.input_ "" { value: name, onChange: setName <?| eventTargetValue }
```

Comment:
Same here: I'd remove the example without the operator. It's never better to use the function name, right?